File: test-env.js
// Simple test script to verify Node.js environment
console.log('Node.js version:', process.version);
console.log('Current directory:', process.cwd());
console.log('Platform:', process.platform);
console.log('Architecture:', process.arch);

// Test file system access
try {
  const fs = require('fs');
  console.log('File system access test:', fs.existsSync('package.json') ? '✅ package.json exists' : '❌ package.json not found');
  
  const packageJson = JSON.parse(fs.readFileSync('package.json', 'utf8'));
  console.log('Project name:', packageJson.name);
  console.log('Node version required:', packageJson.engines?.node || 'Not specified');
  
  console.log('\n✅ Node.js environment test passed!');
} catch (error) {
  console.error('❌ Error during test:', error.message);
  process.exit(1);
}

File: test_env.py
import sys
import os

def test_environment():
    print("=== Python Environment Test ===")
    print(f"Python Version: {sys.version}")
    print(f"Current Working Directory: {os.getcwd()}")
    print("\nEnvironment Variables:")
    for key, value in os.environ.items():
        if 'python' in key.lower() or 'path' in key.lower():
            print(f"{key}: {value}")

if __name__ == "__main__":
    test_environment()
